opentreemap/treemap/instance.py

```diff
@@ -442,6 +442,10 @@ def static_page_names(self):
 
         return names
 
+    def update_geo_rev(self):
+        self.geo_rev += 1
+        self.save()
```

Comment:
To avoid race conditions, consider using [an F expression here](https://docs.djangoproject.com/en/1.7/ref/models/queries/), like we do with `object_caches`